modules/contrib/src/components/Collider.ts

```diff
@@ -1,18 +1,18 @@
 import { Component } from "@piggo-legends/core";
 
 export type ColliderProps = {
-  x: number;
-  y: number;
+  radius: number
+  mass?: number
 }
 
 export class Collider implements Component<"collider"> {
   type: "collider";
 
-  x: number;
-  y: number;
+  radius: number;
+  mass: number;
 
   constructor(props: ColliderProps) {
-    this.x = props.x;
-    this.y = props.y;
+    this.radius = props.radius;
+    this.mass = props.mass ?? 0.001;
```

Comment:
`mass` here doesn't behave how I initially expected. light bodies still move it the same
see https://github.com/liabru/matter-js/issues/890